department-of-veterans-affairs/vets-api
# 79681 - Remove SiS introspect endpoint

## Summary

This PR removes the `/introspect` endpoint for SiS. The tokens we issue are JWTs--not opaque tokens--so an `/introspect` endpoint is not necessary.

## Related issue(s)

https://app.zenhub.com/workspaces/identity-5f5bab705a94c9001ba33734/issues/gh/department-of-veterans-affairs/va.gov-team/79681

## Testing done

* Navigate to the introspect endpoint http://localhost:3000/v0/sign_in/introspect
* Verify not found.

## Screenshots

Not relevant.

## What areas of the site does it impact?

SiS

## Acceptance criteria


## Requested Feedback

None.
